### DISCUSSION TITLE
How to save each channel of an image separately?

### DISCUSSION THREAD

**Discussion by S3Y3D** [kornia/kornia#1298] [Questions] [answered]
Hi guys,
I have an image and I converted its color space from RGB to YCbCr using Kornia.
I want to save each channel (I mean Y, Cb and Cr) separately, something like the image below.
How can I do that?
Thanks
![image](https://user-images.githubusercontent.com/59525413/133880215-ca923d83-3b84-4445-ad25-8805feba8e56.png)

**Comment by edgarriba** [comment, marked as answer]
@S3Y3D you can do as follows:

```python
img = K.color.rgb_to_ycbcr(img_rgb)  # Bx3xHxW
y, cv, cb = img.chunk(dim=-3, chunks=3)
```

**Reply by S3Y3D** [reply]
Thanks 🌻
Unfortunately all channels are in gray, How can I convert the chrominace (Cb and Cr) to be colorful. I mean like the above mentioned image.
The code is:

```
import kornia
import torchvision
from torchvision.utils import save_image

img = torchvision.io.read_image('./kodak-dataset/kodim01.png')/255
img = kornia.color.rgb_to_ycbcr(img)
y, cr, cb = img.chunk(dim=-3, chunks=3)
save_image(y, f'./y.png')
save_image(cr, f'./cr.png')
save_image(cb, f'./cb.png')
```

Output:
![y (Phone)](https://user-images.githubusercontent.com/59525413/133888562-e2668713-8e45-424c-b31b-f4e742ad8729.png)

**Reply by edgarriba** [reply]
You can try with matplotlib that applies colormap to visualize. We don't have such functionality in kornia yet to apply look up tables. Worth to open a ticket  to implement LUT /cc @lferraz had a prototype version for this

**Reply by lferraz** [reply]
At some place we have code for LUTs but here the problem is that you need to create the LUT manually.
Applying the LUT is easy, in the clahe code we have that code.

**Reply by S3Y3D** [reply]
Thanks 🌻
@edgarriba 
@lferraz
